File: src/SOP8/Base/Base.Model/Sop/Component/GridRow.cs
using System;
using dnsDapperDBUtil.DataAccessLayer.IDAL;

namespace Base.Model.Sop.Component
{
	public class GridRow : Table
	{
		public enum Fields { grid_sn, row_no, height };
		public enum WriteFields { grid_sn, row_no, height };

		public int grid_sn { get; set; }
		public int row_no { get; set; }
		public int height { get; set; }

		public static string TableName { get { return "so_compn_grid_row"; } }

		public override string GetTableName()
		{
			return TableName;
		}

		public override string GetPrimaryCondition()
		{
			return string.Format("{0} = {1} and {2} = {3}", Fields.grid_sn, grid_sn, Fields.row_no, row_no);
		}

		public override Type GetFieldType()
		{
			return typeof(Fields);
		}

		public override Type GetWriteFieldType()
		{
			return typeof(WriteFields);
		}

		public void FromCopy(GridRow obj)
		{
			this.grid_sn = obj.grid_sn;
			this.row_no = obj.row_no;
			this.height = obj.height;
		}
	}
}

File: src/SOP8/Egovframework/Base/base-model/src/main/java/egovframework/base/model/sop/component/GridRow.java
package egovframework.base.model.sop.component;

import lombok.*;
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class GridRow {
	public enum Fields { grid_sn, row_no, height }

	private int grid_sn;

	private int row_no;

	private int height;

	public static final String autoIncreaseField = null;
	public static final String tableName = "so_compn_grid_row";
	public static final String[] primaryKeys = { "grid_sn", "row_no" };

}
